new_platforms/vscode-extension/README.md

```diff
@@ -10,61 +10,87 @@ Ensure that the [requirements](#Requirements) are met.
 
 Install the [Microsoft Open Enclave extension](https://marketplace.visualstudio.com/items?itemName=ms-iot.msiot-vscode-openenclave).
 
-## Features
 
-### Create a new Open Enclave solution.
+## Create a new Open Enclave solution.
 
 You can use the `Microsoft Open Enclave: New Open Enclave Solution` command (commands can be found using **F1** or **CTRL-Shift-P**) 
 to create a new Open Enclave solution.  
 
 You will be prompted to:
 
 1. Select a folder for your solution.  
-1. On Linux, you will have the option to create a standalone project or an Azure IoT Edge container 
-    project.  Otherwise, you will only have the option to create an Azure IoT Edge container project.
+1. On Linux, you will have the option to create a [Standalone](#Standalone%20projects) project or an [Azure IoT Edge](#Azure%20IoT%20Edge%20projects) container 
+    project.  Otherwise, you will only have the option to create an [Azure IoT Edge](#Azure%20IoT%20Edge%20projects) container project.
 1. If you choose a container project, you will be prompted to provide your container repository.
 1. You will be prompted to provide a name for your host/enclave.
 
 A new solution will be created in the folder you've selected.  That solution will contain both the host
 and enclave as well as the required EDL file.  If you've chosen to build a container, the host will include
 some code that implements the required Azure IoT Hub communication.
 
+**Note:** On Windows, you can use the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and the 
+Visual Studio Code [Remote-Development extension](https://code.visualstudio.com/docs/remote/remote-overview) to utilize 
+[Standalone](#Standalone%20projects) projects on a Windows desktop.  In this case, you will need to install the [requirements](#Requirements) in
+the subsystem.
+
+## Standalone projects
+
 ### Build your Open Enclave solution.
 
-There are build tasks for both standalone and Azure IoT Edge container projects.  The underlying system used 
-to build is CMake.  
+The underlying system used to build is CMake.  
 
-For a standalone project, there will be configure and build tasks for each target (ARMv7-A and AArch64/ARMv8-A).  The 
+There will be configure and build tasks for each target (ARMv7-A, AArch64/ARMv8-A, and LS1012).  The 
 configure task will invoke CMake to create the required build files.  This is only required to be run once.  
 The build task will do the actual compiling and linking.
 
 1. **F1** or **CTRL-Shift-P**
 1. Select `Tasks: Run Task`
-1. Select `Configure for QEMU (ARMv7-A | AArch64/ARMv8-A)`
-1. Select `Build for QEMU (ARMv7-A | AArch64/ARMv8-A)`
+1. Select `Configure for QEMU (ARMv7-A | AArch64/ARMv8-A | LS1012)`
+1. Select `Build for QEMU (ARMv7-A | AArch64/ARMv8-A | LS1012)`
```

Comment:
It is possible to just run the build task; it'll launch the configure task first.

Reply:
it was when i wrote the docs :) ... maybe not now.

Reply:
removed the config step.